spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/source/SparkBatch.java

```diff
@@ -115,11 +115,11 @@ private String[][] computePreferredLocations() {
   public PartitionReaderFactory createReaderFactory() {
     if (useParquetBatchReads()) {
       int batchSize = readConf.parquetBatchSize();
-      return new SparkColumnarReaderFactory(batchSize);
+      return new SparkColumnarReaderFactory(batchSize, readConf.parquetReaderType());
```

Comment:
This is where we will call `readConf.batchReadConf()` or whatever we want to call it.

Reply:
Changed. Thanks